<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DrILL - Data Reduction for ILL[*]</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="toolbar">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QComboBox" name="modeSelector">
        <property name="toolTip">
         <string>Acquisition mode</string>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="cycleNumber">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Cycle number</string>
        </property>
        <property name="placeholderText">
         <string>Cycle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="experimentId">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Experiment ID</string>
        </property>
        <property name="placeholderText">
         <string>Experiment</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="datadirs">
        <property name="toolTip">
         <string>Manage data directories</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="settings">
        <property name="toolTip">
         <string>Settings window</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="export">
        <property name="toolTip">
         <string>Export settings window</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="load">
        <property name="toolTip">
         <string>Load rundex</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="save">
        <property name="toolTip">
         <string>Save rundex</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QSpinBox" name="nrows">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Number of row</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="addrow">
          <property name="toolTip">
           <string>Add row(s)</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QToolButton" name="deleterow">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>1</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Delete selected row(s)</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="cut">
        <property name="toolTip">
         <string>Cut selected cell(s)</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="copy">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Copy selected cell(s)</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="paste">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Paste cell(s)</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="erase">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Erase selected cell(s)</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QSpinBox" name="increment">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Increment value</string>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>-99</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="fill">
          <property name="toolTip">
           <string>Fill increment of selected cells</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QToolButton" name="help">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Help</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../../../../../images/ILL_logo.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="DrillTableWidget" name="table">
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="footer">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="processRows">
        <property name="toolTip">
         <string>Process selected row(s)</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonProcessGroup">
        <property name="toolTip">
         <string>Process selected group(s)</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonProcessAll">
        <property name="toolTip">
         <string>Process all</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="stop">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Stop processing</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionLoadRundex"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionSave"/>
    <addaction name="actionManageDirectories"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuProcess">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <widget class="QMenu" name="menuRow_s">
     <property name="title">
      <string>&amp;Row(s)</string>
     </property>
     <addaction name="actionAddRow"/>
     <addaction name="actionDelRow"/>
     <addaction name="actionCopyRow"/>
     <addaction name="actionCutRow"/>
     <addaction name="actionPasteRow"/>
    </widget>
    <widget class="QMenu" name="menuColumns_s">
     <property name="title">
      <string>&amp;Columns(s)</string>
     </property>
     <widget class="QMenu" name="menuAddRemoveColumn">
      <property name="title">
       <string>&amp;Add/remove column</string>
      </property>
     </widget>
     <addaction name="menuAddRemoveColumn"/>
    </widget>
    <widget class="QMenu" name="menuCell_s">
     <property name="title">
      <string>Cell(&amp;s)</string>
     </property>
     <addaction name="actionErase"/>
    </widget>
    <addaction name="menuColumns_s"/>
    <addaction name="menuRow_s"/>
    <addaction name="menuCell_s"/>
   </widget>
   <widget class="QMenu" name="menuProcess_2">
    <property name="title">
     <string>&amp;Process</string>
    </property>
    <addaction name="actionProcessRow"/>
    <addaction name="actionProcessGroup"/>
    <addaction name="actionProcessAll_2"/>
    <addaction name="actionStopProcessing"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuProcess"/>
   <addaction name="menuProcess_2"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionLoadRundex">
   <property name="text">
    <string>&amp;Load rundex</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionProcessSelected">
   <property name="text">
    <string>Process selected row(s)</string>
   </property>
  </action>
  <action name="actionProcessAll">
   <property name="text">
    <string>Process selected rows</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>&amp;Save rundex as</string>
   </property>
  </action>
  <action name="actionManageDirectories">
   <property name="text">
    <string>&amp;Manage data directories</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>S&amp;ettings</string>
   </property>
  </action>
  <action name="actionProcessRow">
   <property name="text">
    <string>&amp;Process select row(s)</string>
   </property>
  </action>
  <action name="actionProcessAll_2">
   <property name="text">
    <string>Process all &amp;rows</string>
   </property>
  </action>
  <action name="actionStopProcessing">
   <property name="text">
    <string>&amp;Stop processing</string>
   </property>
  </action>
  <action name="actionCopyRow">
   <property name="text">
    <string>&amp;Copy selected row(s)</string>
   </property>
  </action>
  <action name="actionCutRow">
   <property name="text">
    <string>Cut &amp;selected row(s)</string>
   </property>
  </action>
  <action name="actionPasteRow">
   <property name="text">
    <string>&amp;Paste row(s)</string>
   </property>
  </action>
  <action name="actionErase">
   <property name="text">
    <string>&amp;Erase selected cell(s)</string>
   </property>
  </action>
  <action name="actionAddRow">
   <property name="text">
    <string>&amp;Add a row</string>
   </property>
  </action>
  <action name="actionDelRow">
   <property name="text">
    <string>&amp;Remove selected row(s)</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save &amp;rundex</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>&amp;DrILL help</string>
   </property>
  </action>
  <action name="actiona">
   <property name="text">
    <string>a</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionProcessGroup">
   <property name="text">
    <string>Process selected group(s)</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DrillTableWidget</class>
   <extends>QTableWidget</extends>
   <header>Interface.ui.drill.view.DrillTableWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
